network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/NetworkObjectIndex.java

```diff
@@ -548,7 +548,7 @@ void notifyElementAdded(Identifiable<?> identifiable, Supplier<String> attribute
     void notifyElementAdded(Identifiable<?> identifiable, String attribute, Object newValue) {
```

Comment:
I think method should be renammed to as "notifyPropertyAdded"
In the code it used to notified properties

Reply:
same for the methods after

Reply:
Done